src/test/java/com/puppycrawl/tools/checkstyle/api/BeforeExecutionFileFilterSetTest.java

```diff
@@ -115,4 +116,10 @@ public void testUnmodifiableSet() {
             () -> filterSet.getBeforeExecutionFileFilters().add(filter));
     }
 
+    @Test
```

Comment:
Please make a comment on method with details on why we kill this mutation with pure unit test.

Reply:
ok. Missed that while fixing an error probably.

Reply:
@nrmancuso 
Also jupiter assertion api have been used here. Is that fine?